change/office-ui-fabric-react-2019-12-27-13-39-03-reactdnd.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Layer: add touch events to filtering to stop those that are bubbling up",
+  "packageName": "office-ui-fabric-react",
+  "email": "email not defined",
```

Comment:
A bit odd, is email set up in your git config?

Reply:
I thought I correctly set it up I'll redo git config.